.github/workflows/admin-sample.cd.yml

```
@@ -69,6 +69,57 @@
         path: ${{env.DOTNET_ROOT}}/server
         include-hidden-files: true # Required for wwwroot/.well-known folder
 
+  deploy_blazor_wasm_standalone:
+    name: build blazor wasm standalone
+    runs-on: ubuntu-24.04
+
+    steps:
+    
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)